packages/react-reconciler/src/__tests__/ReactIncrementalSideEffects-test.internal.js

```diff
@@ -599,7 +601,7 @@ describe('ReactIncrementalSideEffects', () => {
     // render some higher priority work. The middle content will bailout so
     // it remains untouched which means that it should reuse it next time.
     ReactNoop.render(<Foo text="foo" step={1} />);
-    expect(ReactNoop).toFlushAndYield(['Foo', 'Content', 'Bar', 'Bar']);
+    expect(Scheduler).toFlushAndYield(['Foo', 'Content', 'Bar', 'Bar']);
```

Comment:
This seems very confused to me because `shouldComponentUpdate` returns `false` but the renderer still choose to render it(the `Content` component) which is very different to the DOM renderer, any details would you mind to share?